README.rst

```diff
@@ -0,0 +1,43 @@
+Mozzarella
+==========
+
+This website is a website for clubs and student organizations.
+
+Mozzarella It is written in Python and uses the powerful web framework
+`TurboGears`.
+
+.. TurboGears: http://turbogears.com/
```

Comment:
See the reStructuredText specification on how to create links::

    powerful web framework TurboGears_.

    .. _TurboGears: http://turbogears.org

http://docutils.sourceforge.net/docs/user/rst/quickref.html

Reply:
Site is down:

"Project web is currently offline pending the final migration of its data to our new datacenter."